src/Employer/Employer.Web/Extensions/VacancySummaryExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Data;
+using Esfa.Recruit.Employer.Web.Configuration.Routing;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.Employer;
+
+namespace Esfa.Recruit.Employer.Web.Extensions
+{
+    public static class VacancySummaryExtensions
+    {
+        public static string GetLinkText(this VacancySummary vacancySummary)
+        {
+            switch (vacancySummary.Status)
+            {
+                case VacancyStatus.Referred:
+                    return "Edit and resubmit";
+                case VacancyStatus.Draft:
+                    return "Edit and submit";
+            }
+
+            return "View";
+        }
+
+        public static string GetLinkRoute(this VacancySummary vacancySummary)
+        {
+            if (vacancySummary.Status == VacancyStatus.Live || vacancySummary.Status == VacancyStatus.Closed)
```

Comment:
Is the Closed because of the edit dates vacancy because we cannot edit or perform any actions on Closed vacancies yet. I think maybe checking the status is live is all that is needed.

Reply:
👍 Manage is just redirecting to View for non-live vacancies so this makes sense